yarn.lock

```diff
@@ -2717,9 +2717,9 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@openmrs/esm-api@npm:5.5.1-pre.1659":
```

Comment:
Thanks @kb019 for this, i think it would have been great to separate the bump from fixing the 400 error